salt/redmine/init.sls

```diff
@@ -0,0 +1,88 @@
+{% from 'lib.sls' import apache, create_user %}
+
+include:
+  - apache
+  - apache.modules.passenger
+  - mysql
+  - rvm
+
+{{ create_user(pillar.redmine.user) }}
+
+/home/{{ pillar.redmine.user }}/public_html:
+  file.directory:
+    - user: {{ pillar.redmine.user }}
+    - group: {{ pillar.redmine.user }}
+    - require:
+      - user: {{ pillar.redmine.user }}_user_exists
+
+redmine dependencies:
+  pkg.installed:
+    - pkgs:
```

Comment:
According to the docs we also need `build-essential` https://redmine.org/projects/redmine/wiki/HowTo_Install_Redmine_50x_on_Ubuntu_2004_with_Apache2

Reply:
The test Redmine install I have is working fine without this package, I am not sure what requires it in their install process.
Happy to install `build-essential` or to leave it and install when we need it.

Reply:
We can leave it off.